common/static/common/js/discussion/views/discussion_thread_list_view.js

```diff
@@ -370,20 +376,16 @@
             };
 
             DiscussionThreadListView.prototype.renderThread = function(thread) {
-                var content, unreadCount;
-                content = $(_.template($('#thread-list-item-template').html())(thread.toJSON()));
-                unreadCount = thread.get('unread_comments_count') + (thread.get('read') ? 0 : 1);
-                if (unreadCount > 0) {
-                    content.find('.forum-nav-thread-comments-count').attr(
-                        'data-tooltip',
-                        edx.StringUtils.interpolate(
-                            ngettext('{unread_count} new comment', '{unread_count} new comments', unreadCount),
-                            {unread_count: unreadCount},
-                            true
-                        )
+                var threadCommentCount = thread.get('comments_count'),
+                    threadUnreadCommentCount = thread.get('unread_comments_count'),
+                    neverRead = !thread.get('read') && threadUnreadCommentCount == threadCommentCount,
+                    context = _.extend(
+                        {
+                            neverRead: neverRead
+                        },
+                        thread.toJSON()
                     );
-                }
-                return content;
+                return $(this.threadListItemTemplate(context).toString());
```

Comment:
Haven't seen this pattern before - what happens / what's the utility of wrapping a `String` in a call to `$()`?